modules/pilotxBidAdapter.js

```diff
@@ -0,0 +1,147 @@
+      return false
+    }
+    return !!(bid.params.placementId || (bid.params.member && bid.params.invCode));
+  },
+  /**
+         * Make a server request from the list of BidRequests.
+         *
+         * @param {validBidRequests[]} - an array of bids
+         * @return ServerRequest Info describing the request to the server.
+         */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    let payloadItems = {};
+    let sizes = [];
+    validBidRequests.forEach(bidRequest => {
+      let placementId = this.setPlacementID(bidRequest.params.placementId)
+      payloadItems[placementId] = {}
+      if (bidRequest.sizes.length > 0) {
+        if (Array.isArray(bidRequest.sizes[0])) {
+          for (let i = 0; i < bidRequest.sizes.length; i++) {
+            sizes[i] = [(bidRequest.sizes[i])[0], (bidRequest.sizes[i])[1]]
+          }
+        } else {
+          sizes[0] = [bidRequest.sizes[0], bidRequest.sizes[1]]
+        }
+        payloadItems[placementId]['sizes'] = sizes
+      }
+      if (bidRequest.mediaTypes != null) {
+        for (let i in bidRequest.mediaTypes) {
+          payloadItems[placementId][i] = {
+            ...bidRequest.mediaTypes[i]
+          }
+        }
+      }
+      let consentTemp = ''
+      let consentRequiredTemp = false
+      if (bidderRequest && bidderRequest.gdprConsent) {
+        consentTemp = bidderRequest.gdprConsent.consentString
+        // will check if the gdprApplies field was populated with a boolean value (ie from page config).  If it's undefined, then default to true
+        consentRequiredTemp = (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') ? bidderRequest.gdprConsent.gdprApplies : true
+      }
+
+      payloadItems[placementId]['gdprConsentString'] = consentTemp
+      payloadItems[placementId]['gdprConsentRequired'] = consentRequiredTemp
+      payloadItems[placementId]['bidId'] = bidRequest.bidId
+    });
+    const payload = payloadItems;
+    const payloadString = JSON.stringify(payload);
+    return {
+      method: 'POST',
+      url: ENDPOINT_URL,
+      data: payloadString,
+    };
+  },
+  /**
+         * Unpack the response from the server into a list of bids.
+         *
+         * @param {ServerResponse} serverResponse A successful response from the server.
+         * @return {Bid[]} An array of bids which were nested inside the server.
+         */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const serverBody = serverResponse.body;
+    const bidResponses = [];
+    if (serverBody.mediaType == 'banner') {
+      const bidResponse = {
+        requestId: serverBody.requestId,
+        cpm: serverBody.cpm,
+        width: serverBody.width,
+        height: serverBody.height,
+        creativeId: serverBody.creativeId,
+        currency: serverBody.currency,
+        netRevenue: false,
+        ttl: serverBody.ttl,
+        ad: serverBody.ad,
+        mediaType: 'banner',
+        meta: {
+          mediaType: 'banner'

```

Comment:
Can you please add support for `advertiserDomains` meta tag in this new adapter PR, adomain support is now required for all new adapter submissions.